packages/babel-plugin-transform-typescript/test/fixtures/namespace/empty-removed/output.mjs

```diff
@@ -0,0 +1,37 @@
+let a;
+(function (_a) {
+  let c;
+  (function (_c) {
+    var x;
+  })(c || (c = {}));
+})(a || (a = {}));
+let WithTypes;
+(function (_WithTypes) {
+  let d;
+  (function (_d2) {})(d || (d = {}));
+})(WithTypes || (WithTypes = {}));
```

Comment:
This is weird but it matches TS: https://www.typescriptlang.org/play?#code/HYQwtgpgzgDiDGEAEB1AlgFwBYBUCeM0SA3gFBJKiSwLIglIYHICCSAvEgIxIC+5lcNDiIkAIwZpgGCACcAZrSQAhErz4CqwpfElgYAe1kYkAYQ5J6-ClpqiAJg3sR4AGxCzkbkFChIAImoa-EA